src/components/CreatePostModal/index.tsx

```diff
@@ -171,15 +179,16 @@ const CreatePostModal = ({ visible, onClose, userId, onSelect }: IModal) => {
             const yOffset = nativeEvent.contentOffset.y;
             const contentHeight = nativeEvent.contentSize.height;
             const scrollViewHeight = nativeEvent.layoutMeasurement.height;
-            const isNearBottom = contentHeight - yOffset <= scrollViewHeight * 1.7; // Adjust the multiplier as needed
+            const isNearBottom =
+              contentHeight - yOffset <= scrollViewHeight * 1.7; // Adjust the multiplier as needed
 
             if (isNearBottom) {
               handleEndReached();
             }
           }}
           scrollEventThrottle={16} // Adjust as needed
         >
-          {communities.map(item=>renderCommunity({item}))}
+          {communities.map((item, index) => renderCommunity({ item, index }))}
```

Comment:
do we really need an index

Reply:
Sometime, we cannot get postId and sometime these IDs are identical somehow.
And I cannot control backend code as well.
That's why I handled in Front End and I used index as key.

Whatever I will revert it due to performance. xD